pkg/cloudprovider/providers/providers.go

```diff
@@ -1,3 +1,5 @@
+// +build !nolegacyproviders
```

Comment:
We shouldn't need the build tags in the openstack provider files if we have it here or am I missing something?

Reply:
If we don't put the tags in the provider files then we can't guarantee nothing is using them, in fact unfortunately not everything goes through the cloud provider interface :(
This guarantees they are compiled out.

(Unless you mean specifically this provider should be ignored?)

Reply:
>  we can't guarantee nothing is using them

Good point :)